Lacrit/assignment1/q1.cpp

```diff
@@ -0,0 +1,49 @@
+#include <iostream>
+#include <unordered_map>
+#include <boost/algorithm/string.hpp>
+
+
+// input : string a , string b || output : bool 
+// determine if one is an anagram of another 
+	
+std::unordered_map<char, int> count_occurences(std::string& a, const bool c)
```

Comment:
Parameters can use better names :)
Specially bools, that could be one thing or the exact opposite.